### DISCUSSION TITLE
Dark mode in CSS Modules

### DISCUSSION THREAD

**Discussion by juan-manuel-alberro** [mantinedev/mantine#4833] [Q&A] [answered]
Hi all,

I migrated to v7 and I'd like to know what's the best practice to deal with dark mode styling using CSS Modules.
The documentation doesn't mention anything related to dark mode in v7.

Thanks

**Comment by juan-manuel-alberro** [comment, marked as answer]
I just found this example

https://github.com/mantinedev/next-pages-template/blob/master/components/Welcome/Welcome.module.css#L2

```css
  color: light-dark(var(--mantine-color-black), var(--mantine-color-white));
```
That's not mentioned in the documentation so I'll leave it here just in case it might be useful for someone else.

**Reply by rtivital** [reply]
https://mantine.dev/styles/postcss-preset/
